packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -361,6 +363,10 @@ const AccountTransferForm = observer(
             );
         };
 
+        if (is_transfer_confirm) {
+            return <AccountTransferReceipt onClose={onClose} />;
+        }
```

Comment:
Render `<AccountTransferReceipt />` before anything else to keep the same hierarchy as before.